postgres/9.6/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM postgres:9.6
+
+MAINTAINER Debezium Community
+
+# Install the packages which will be required to get everything to compile
+RUN apt-get update && apt-get install -y build-essential pkg-config git postgresql-server-dev-9.6 software-properties-common
+
+# The plugin requires protobuf-c at least 1.1 but the stable Debian repo has an older version, so we need to add the Debian testing repo which has the correct version
+RUN add-apt-repository "deb http://ftp.debian.org/debian testing main contrib" && apt-get update
+ 
+# Install the protobuf-c dependency for the plugin
+RUN apt-get install -y libprotobuf-c-dev=1.2.1-1+b1
+ 
+# Compile the plugin from sources and install it
+# TODO: the plugin should probably be forked in the DBZ repository
+RUN git clone https://github.com/hchiorean/decoderbufs
+RUN cd /decoderbufs && make && make install && cd /
```

Comment:
Should we "release" the code by making a tag, so the this build can check out that tag? Also, because of how Docker filesystems are layered, we probably want to use a single `RUN` command that also cleans up any intermediary files created by the build _and_ removes the local git repository. By doing all these steps on a single line, the only files that go into the new layer will be those that exist at the end of the line. Thoughts?